packages/legend-studio-app/src/index.tsx

```diff
@@ -31,6 +32,7 @@ export class LegendStudioWebApplication {
       .withPresets([
         ...getLegendGraphExtensionCollection(),
         new QueryBuilder_LegendStudioPreset(),
+        new ELMorphir_LegendStudioPreset(),
```

Comment:
Let's follow the other ones down there, we should really only import `ELMorphir_LegendStudioPlugin` and leave other parts to `getLegendGraphExtensionCollection()`